src/test/java/seedu/address/logic/PasswordTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+//@@author jelneo
+public class PasswordTest {
+
+    @Test
+    public void isValidPasswordLength() {
+        // password does not meet length requirements
+        assertFalse(Password.isValidPasswordLength(""));
+        assertFalse(Password.isValidPasswordLength("_"));
+        assertFalse(Password.isValidPasswordLength("pwd_1"));
+
+        // password meets length requirement
+        assertTrue(Password.isValidPasswordLength("abcde12345"));
+        assertTrue(Password.isValidPasswordLength("ABCE1234!@#"));
+    }
+
+    @Test
+    public void isValidPassword() {
+        // blank password
+        assertFalse(Password.isValidPasswordLength(""));
+        assertFalse(Password.isValidPasswordLength(" "));
```

Comment:
Should these two be .isValidPassword instead?

Reply:
renamed